### PR TITLE
Add docs for certificate manager

### PR DESCRIPTION
**Pull request by madebysid**

### CODE REVIEW

**Comment by madebysid** [comment]
@shamasis @czardoz @numaanashraf @SamvelRaja Can you go through this as well?

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+page_id: "working_with_certificates"
+tags:
+  - "native"
+  - "mac"
+  - "windows"
+  - "linux"
+warning: false
+---
+
+Postman's [native apps](http://www.getpostman.com/apps) provide a Certificate Manager that lets you view certificates that are associated with each domain.
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: SSL Certificate Manager

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To open the Certificate Manager, click the wrench icon on the top right (or use the shortcut `Cmd/Ctrl + ,`), choose `Settings`, then switch over to the `Certificates` tab in the popover.
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: remove "in the popover". Popover does not mean anything to the many.

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+![](https://cloud.githubusercontent.com/assets/7689783/19720908/bbbd8d9c-9b8d-11e6-9286-8e8ba12d9c68.png)
+
+### Add a Certificate
+
+Too add a new certificate, click on `Add Certificate` in the Certificate Manager.
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: Too?

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+In the `Host` field, enter your domain (without the protocol).
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: enter the domain (without protocol) of the request URL for which you want to use the certificate.

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+You can also specify a custom port to associate with this domain in the `Port` field. This is optional.
+
+Choose your certificate file in the field called `CRT file`. Currently, we only support the CRT format. Support for other formats (like PFX) will come soon.
+
+Choose your certificate key file in the field called `KEY file`.
+
+If you set a passphrase while generating the certificate, you'll need to supply the passphrase in the `Passphrase` field. This is optional.
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: If you **have** set

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+![](https://cloud.githubusercontent.com/assets/7689783/19721340/7a071024-9b8f-11e6-97d2-814aa3075c80.png)
+
+**NOTE:** You cannot have multiple certificates set for the same domain. If you have multiple ones set, only the last one set will be used.
+
+### Use a Certificate
+
+You do not have to perform any extra steps to use a certificate if it is added to the `Certificate Manager`. If you make a request to the set domain, the certificate will automatically be sent with the request. (Provided you make the request over HTTPS)
+
+We can verify this. To do so, open up your Postman Console (Cmd/Ctrl + Alt + C). You can read more about it [here](http://blog.getpostman.com/2016/08/26/the-postman-console/). A new window should open up.
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: Everywhere it is "you", why "we" here?

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Now, let's send a request to `https://echo.getpostman.com/get`, keeping the Postman Console open. Notice we're using `https` to make sure the certificate is sent. Once the response arrives, switch over to the Postman Console. You should see your request there. If you expand your request, you should be able to see which certificate was sent along with the request.
+
+![](https://cloud.githubusercontent.com/assets/7689783/19721699/0ccdeada-9b91-11e6-98af-eb08f8e68f5b.png)
+
+### Remove a Certificate
+
+To remove a certificate, you can use the `Remove Certificate` next to the certificate in the `Certificate Manager`
```
Comment (shamasis) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: `Remove Certificate` button

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md (hunk 5 above)
Comment (madebysid) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: How about "If you used a passphrase while generating the certificate,"

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
+---
+category: "docs"
+title: "Working with certificates"
```
Comment (numaanashraf) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: Working with client certificates

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md (hunk 2 above)
Comment (czardoz) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: I would just say something like:

    To open the Certificate Manager, open [Postman Settings](https://www.getpostman.com/docs/settings), then switch over to the `Certificates` tab.

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_client_certificates.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  - "native"
+  - "mac"
+  - "windows"
+  - "linux"
+warning: false
+---
+
+Postman's [native apps](http://www.getpostman.com/apps) provide a SSL Certificate Manager that lets you view / set client certificates on a per domain basis.
+
+To open the Certificate Manager, click the wrench icon on the top right (or use the shortcut `Cmd/Ctrl + ,`), choose `Settings`, then switch over to the `Certificates` tab.
```
Comment (czardoz) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_client_certificates.md]: I would just say something like:

    To open the Certificate Manager, open [Postman Settings](https://www.getpostman.com/docs/settings), then switch over to the `Certificates` tab.

**Comment by madebysid** [comment]
@shamasis @numaanashraf Addressed comments.

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_client_certificates.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+### Add a Certificate
+
+To add a new certificate, click on `Add Certificate` in the Certificate Manager.
+
+In the `Host` field, enter enter the domain (without protocol) of the request URL for which you want to use the certificate.
+
+You can also specify a custom port to associate with this domain in the `Port` field. This is optional.
+
+Choose your certificate file in the field called `CRT file`. Currently, we only support the CRT format. Support for other formats (like PFX) will come soon.
```
Comment (czardoz) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_client_certificates.md]: Assert that users should choose their _Client_ certificate file.

Comment (madebysid) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_client_certificates.md]: I'll add that assertion in other places too.
Also adding paragraph demonstrating how to export a certificate from the OSX keychain.

**Comment by madebysid** [comment]
@czardoz Addressed comments. Will add demo of exporting from the macOS keychain once we support PFX.

File: _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+Once you are finished, click on `Add`.
+
+![](https://cloud.githubusercontent.com/assets/7689783/19721093/75d764c8-9b8e-11e6-85c2-feff9eea4345.png)
+
+Once your certificate is added, it should appear in the Certificate Manager list.
+
+![](https://cloud.githubusercontent.com/assets/7689783/19721340/7a071024-9b8f-11e6-97d2-814aa3075c80.png)
+
+**NOTE:** You cannot have multiple certificates set for the same domain. If you have multiple ones set, only the last one set will be used.
```
Comment (abhijitkane) [inline _posts/02_sending_and_viewing_responses/2016-07-11-working_with_certificates.md]: Mutually contradictory. Say `you should not set multiple .... If you have multiple ones set, ...`

**Pull request merged**